src/providers/proxy/proxy_id.c

```diff
@@ -1488,6 +1488,152 @@ static int get_initgr(TALLOC_CTX *mem_ctx,
     return ret;
 }
 
+static int compare_group_members(struct proxy_id_ctx *ctx,
+                       TALLOC_CTX *tmp_ctx,
+                       struct sss_domain_info *dom,
+                       long int num_gids,
+                       gid_t *gids,
+                       char *db_groupname,
```

Comment:
Most of pointers should be pointers to const, right?

Reply:
I've changed them. Can you review them in the new revision?

Reply:
Looks ok, thanks.